contrib/externalMatchfinder/matchfinder.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Yann Collet, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under both the BSD-style license (found in the
+ * LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ * in the COPYING file in the root directory of this source tree).
+ * You may select, at your option, one of the above-listed licenses.
+ */
+
```

Comment:
Definitely worth having a short introduction paragraph to describe what this unit is about (simple external matchfinder for test and illustration purposes).

Reply:
Added contrib/externalMatchfinder/README.md